my-sms/db/migrate/20210208041633_create_titles.rb

```diff
@@ -0,0 +1,34 @@
+class CreateTitles < ActiveRecord::Migration
+  HONORIFICS = %w[Mr Mrs Miss Ms Master]
```

Comment:
Perhaps we can move this to the title model so it can be used elsewhere in the app

Reply:
I feel like that would create multiple sources of truth for what the available titles are. I'm happy to move it if you disagree.

Reply:
Yes, you are correct - let's leave it here